src/quickstart/provider/aws/boot.md

```diff
@@ -1,263 +1,364 @@
-controller-manager   Healthy   ok
-etcd-0               Healthy   {"health": "true"}
-etcd-1               Healthy   {"health": "true"}
-Cluster validation succeeded
-Done, listing cluster services:
-
-Kubernetes master is running at https://52.9.206.49
-Elasticsearch is running at https://52.9.206.49/api/v1/proxy/namespaces/kube-system/services/elasticsearch-logging
-Heapster is running at https://52.9.206.49/api/v1/proxy/namespaces/kube-system/services/heapster
-Kibana is running at https://52.9.206.49/api/v1/proxy/namespaces/kube-system/services/kibana-logging
-KubeDNS is running at https://52.9.206.49/api/v1/proxy/namespaces/kube-system/services/kube-dns
-kubernetes-dashboard is running at https://52.9.206.49/api/v1/proxy/namespaces/kube-system/services/kubernetes-dashboard
-Grafana is running at https://52.9.206.49/api/v1/proxy/namespaces/kube-system/services/monitoring-grafana
-InfluxDB is running at https://52.9.206.49/api/v1/proxy/namespaces/kube-system/services/monitoring-influxdb
-
-Kubernetes binaries at /Users/jhansen/p/docs/kubernetes/cluster/
-You may want to add this directory to your PATH in $HOME/.profile
-Installation successful!
+
+All instances created by `kops` will be built within ASG (Auto Scaling Groups),
+which means each instance will be automatically monitored and rebuilt by AWS if
+it suffers any failure.
+
+#### Customize Cluster Configuration
+
+Now we have a cluster configuration, we can look at every aspect that defines
+our cluster by editing the description.
+
+```bash
+kops edit cluster ${NAME}
 ```
 
-## Items of note!
+This opens your editor (as defined by $EDITOR) and allows you to edit the
+configuration.  The configuration is loaded from the S3 bucket we created
+earlier, and automatically updated when we save and exit the editor.
 
-A few things to note! Your Kubernetes master is now up and running and we are ready to install Deis Workflow. If you
-need to access the Kubernetes master the default username is `admin` and the ssh key lives at `~/.ssh/kube_aws_rsa`.
+We'll leave everything set to the defaults for now, but the rest of the `kops`
+documentation covers additional settings and configuration you can enable.
 
+#### Build the Cluster
+
+Now we take the final step of actually building the cluster.  This'll take a
+while.  Once it finishes you'll have to wait longer while the booted instances
+finish downloading Kubernetes components and reach a "ready" state.
+
+```bash
+kops update cluster ${NAME} --yes
 ```
-$ ssh -i ~/.ssh/kube_aws_rsa admin@52.9.206.49
 
-Welcome to Kubernetes v1.3.6!
+#### Use the Cluster
+
+Remember when you installed `kubectl` earlier? The configuration for your
+cluster was automatically generated and written to `~/.kube/config` for you!
+
+A simple Kubernetes API call can be used to check if the API is online and
+listening. Let's use `kubectl` to check the nodes.
 
-You can find documentation for Kubernetes at:
-  http://docs.kubernetes.io/
+```bash
+kubectl get nodes
```

Comment:
This is a good point, and kops validation is a poor implementation of this. Let's suggest `kubectl cluster-info` I like that more.